FBWinSDK/FBWinSDK/FBWinSDK.Shared/FacebookSession.cpp

```
@@ -852,15 +853,27 @@ IAsyncOperation<FBResult^>^ FBSession::LoginAsync(
         return create_task([=]() -> FBResult^
         {
             FBResult^ result = nullptr;
-
-            task<FBResult^> authTask = TryLoginViaWebView(parameters);
-            result = authTask.get();
-            if (!result)
+            task<FBResult^> authTask;
+            switch (behavior)
```

This switch statement is missing a `default` label, it would be good to have one that does some sort of logging or debug print statement if it gets hit, in case the `SessionLoginBehavior` enum class ever gets changed the sdk won't silently fail. `OutputDebugString` should work for this.